test/browsercontext.spec.js

```diff
@@ -651,4 +651,36 @@ describe('Events.BrowserContext.Page', function() {
     ]);
     await context.close();
   });
+  it.fail(CHROMIUM || WEBKIT)('should work with Shift-clicking', async({browser, server}) => {
```

Comment:
Do these two tests somehow differ from those in popup.spec.js below?

Reply:
This one waits for the 'page' event, the other waits for 'popup' event.